modules/cnf-performance-profile-creator-arguments.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+// Epic CNF-792 (4.8)
+// * scalability_and_performance/cnf-create-performance-profiles.adoc
+
+
+[id="performance-profile-creator-arguments_{context}"]
+=  Performance Profile Creator arguments
+
+.Performance Profile Creator arguments
+["cols="30%,70%",options="header"]
+|===
+| Argument | Description
+
+| `disable-ht`
+a|Disable hyperthreading.
+
+Possible values: `true` or `false`.
+
+Default: `false`.
+
+[WARNING]
+====
+If this argument is set to `true` you should not disable hyperthreading in the BIOS. Disabling hyperthreading is accomplished with a kernel command line argument.
+====
+
+| `info`
+a| This captures cluster information and is used in discovery mode only. Discovery mode also requires the `must-gather-dir-path`. If any other arguments are set they are ignored.
```

Comment:
Since you have a "the" in front of `must-gather-dir-path`, you need "parameter" or "argument" after it or something. Or just remove the "the".